src/hooks/safe-apps/useQueryFilter.ts

```diff
@@ -0,0 +1,57 @@
+import { type Dispatch, type SetStateAction, useEffect } from 'react'
+import { useRouter } from 'next/router'
+import { getCategoryOptions } from '@/components/safe-apps/SafeAppsFilters'
+import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
+
+// TODO: Add search and batch filter as well
```

Comment:
Please don't add todos that will never be done.

Reply:
In that case I would suggest we rename this hook to something like `useCategoryFilter`

I could see it in the future that we want to link to a specific search query or with apps pre-selected that support batching.

Reply:
`useCategoryFilter` sounds much clearer to me.

To clarify what I meant: code comments are not the right place for feature ideas and it's not a backlog.